static/js/analytics.js

```diff
@@ -64,24 +76,51 @@ const composeStatisticsFromSingleUserJson = (json) => {
     .slice(0, 19)
     .replace("T", " ");
   const taggedFriends = Array.from(new Set(taggedFriendsFromJson(json)));
-  pageHtml += `<h2>${lastUserName}</h2>`;
-  pageHtml += `<p>Last fetched story: ${lastFetchedStoryTime}</p>`;
-  pageHtml += `<p>N: ${nStories} Stories | N: ${nPics} Pictures | N: ${nVideos} Videos</p>`;
-  pageHtml += `<canvas width="100%" id="chart-media"></canvas>`;
-  pageHtml += `<canvas width="100%" id="chart-hourly-freq"></canvas>`;
 
-  if (NGPSLocations > 0) {
-    pageHtml += `<p>Geotagged ${NGPSLocations} times</p>`;
+  const userDetails = {
```

Comment:
I'd rename userDetails to something clearer, It's not details about an user but about the current processed results.

eg: userDetails -> userSelectedStoriesInfo

Reply:
Done